protocol/migrations/sqlite/1698396318_profile_showcase_contacts.up.sql

```diff
@@ -0,0 +1,9 @@
+DROP TABLE profile_showcase_contacts;
+
+CREATE TABLE IF NOT EXISTS profile_showcase_contacts (
```

Comment:
I guess `IF NOT EXISTS` is not needed if we drop the table just one line earlier 😄